frontend/public/components/ingress.jsx

```diff
@@ -58,10 +58,10 @@ const TLSCert = ({ ingress }) => {
 };
 
 const tableColumnClasses = [
-  classNames('col-md-3', 'col-sm-4', 'col-xs-6'),
-  classNames('col-md-3', 'col-sm-4', 'col-xs-6'),
-  classNames('col-md-3', 'col-sm-4', 'hidden-xs'),
-  classNames('col-md-3', 'hidden-sm', 'hidden-xs'),
+  '',
+  '',
+  'pf-m-hidden pf-m-visible-on-md',
+  'pf-m-hidden pf-m-visible-on-md',
```

Comment:
Currently we show the `Hosts` column at browser sizes > 992px which is `pf-m-visible-on-lg`  
```suggestion
  'pf-m-hidden pf-m-visible-on-md',
  'pf-m-hidden pf-m-visible-on-lg',
```

Reply:
@sg00dwin Updated